R/pit.R

```diff
@@ -238,20 +239,30 @@ pit <- function(true_values,
 #'
 #' @param PIT_samples A vector with the PIT values of size n
 #' @param num_bins the number of bins in the PIT histogram.
+#' @param caption provide a caption that gets passed to the plot
 #' If not given, the square root of n will be used
 #' @return vector with the scoring values
-#' @importFrom graphics hist
+#' @importFrom ggplot2 ggplot aes xlab ylab geom_histogram stat
```

Comment:
If you really want to go crazy with keeping down the deps you could make this plot optional and have it as a suggests (like EpiSoon etc). I think imported is fine though as this seems like fairly core functionality (and I assume future plots will be coming)